uint16 speedLeft
int8 dirLeft
uint16 speedRight
int8 dirRight
